### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by NthabisengLekolwana**
created js file

### CODE REVIEW

**Comment by XolaniMba** [comment]
the whole chatbot looks nice, and the icon used for the chatbot looks good..adding more responses

**Comment by ManeoMalatsi** [comment]
On your chatbot header, please try to come up with another way to ask people to connect with you. Refrain using the word "please"

**Comment by neoThobela** [comment]
the code is easy to understand add more info about your portfolio

**Review by neoThobela — COMMENTED** [review]
please try to add more information ob your chatbot about your portfolio

File: MyPortfolio/portfolio/index.html [hunk 1]
```diff
@@ -8,67 +8,148 @@
...
         <h3>Experience</h3>
-        <p>asdsdfghjklkjhgfdsdfghjkhgfdsfghjklkjhgfdsdfghjkjhgfdsfghj</p>
+        <p class="blackText">I did a 10 months  IT internship at Phalaphala Consulting<br> 
+          Management, based at Moretele Local Municipality. <br>
+          I was doing support,mainly helpdesk.
+            </p>
       </div>
       <div class="presently">
         <h3>Education</h3>
-        <p>asdfghjkkloiuytrewqazxcvbnmjujnhyhbgtgvfrfcaqsxswdcrfthjhgfdrtjds</p>
```
Comment (kgaugelomokgalaka) [inline MyPortfolio/portfolio/index.html:46]: please remove this part

**Review by kgaugelomokgalaka — COMMENTED** [review]
your code is good 
please just fix the above highlited comment

**Review by kgaugelomokgalaka — APPROVED** [review]
it looks good now

**Comment by NthabisengLekolwana** [comment]
> please try to add more information ob your chatbot about your portfolio

@neoThobela , done.

**Comment by NthabisengLekolwana** [comment]
> On your chatbot header, please try to come up with another way to ask people to connect with you. Refrain using the word "please"

@ManeoMalatsi I removed the 'please'

**Review by neoThobela — APPROVED** [review]

**Comment by neoThobela** [comment]
@NthabisengLekolwana thank you

**Review by XolaniMba — APPROVED** [review]

**Pull request merged**